frontend/src/views/tasks/TaskDetailView.vue

```diff
@@ -932,7 +939,7 @@ async function removeRepeatAfter() {
 	task.value.repeatAfter.amount = 0
 	task.value.repeatMode = TASK_REPEAT_MODES.REPEAT_MODE_DEFAULT
 	await saveTask()
-}
+}~
```

Comment:
This seems like it should not be here

```suggestion
}
```